yew/src/virtual_dom/mod.rs

```diff
@@ -38,6 +38,48 @@ pub use self::vtag::VTag;
 #[doc(inline)]
 pub use self::vtext::VText;
 
+cfg_if! {
```

Comment:
Same as my other small nit. This doesn't really matter, but it means that `rustfmt` can get at this code (which it couldn't otherwise).